examples/api/lib/widgets/heroes/hero.1.dart

```diff
@@ -18,8 +18,9 @@ class HeroApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
-      home: HeroExample(),
+    return MaterialApp(
+      theme: ThemeData(useMaterial3: true),
```

Comment:
Just double check if it is intended to set `useMaterial3` to true in this example:)

Reply:
Yes. All of the examples should use M3. That will happen automatically once we change the default and so changes like this can be withdrawn once the `useMaterial3: true` change lands.

Reply:
I see. Thanks!